docs/source/openshift/exposing-services.adoc

```diff
@@ -0,0 +1,63 @@
+[[exposing-services]]
+= Exposing services
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[overview]]
+== Overview
+
+There are several ways you can expose your service once deployed on OpenShift.
+The following sections describes the various alternatives and when to use them.
+
+[[application-routes]]
+== Routes
+
+Provided you are deploying some form of web application, the most common way to expose it is by a so called link:https://docs.openshift.org/latest/dev_guide/routes.html[route].
```

Comment:
Use of active voice in the first part of the sentence is missing. Why are we using 'so called' it could imply that it is 'Incorrectly or falsely termed' as routes.